resources/splunk_app.rb

```diff
@@ -33,11 +31,11 @@ def app_path
   end
 
   def config_scope
-    'local'
+    @config_scope ||= (resource_name == :splunk_app_custom ? 'default' : 'local')
   end
 
-  def splunk_app_path
-    Pathname.new(install_dir).join('etc/apps')
+  def app_cache_path
+    @app_cache_path ||= Pathname.new(Chef::Config['file_cache_path']).join('splunk_ingredient/old_apps')
```

Comment:
I'm not sure there's an established style regarding pathnames, but I feel that situations including `Pathname.new(...)` or ending with method calls should continue to use methods such as `.join()`, and concatenation (`+`) is to be used with simple references. It is rather inconsistent in this file, and I will remedy this, but I'm not certain this is a specific case that should be changed.